README.md

```diff
@@ -208,3 +208,6 @@ Helpful information about Dapps.
 
 <br>
 
+### Note from practical use
+- [Note of setting up testing environment](https://github.com/BlockchainLabsNZ/awesome-solidity/blob/master/Note_of_setting_up_testing_environment.md) - A note of how to set up testing environment and avoid annoying errors (on Windows10). Including a few regular tools: testrpc, truffle, mocha.js,truffle flattener, gas-reporter, coveralls, sol-function-profiler, Parity
```

Comment:
1. Why not just a "Windows 10 test environment setup"?
2. I think it is better to give relative paths, e.g: [...](Note_of_setting_up_testing_environment.md)
3. "annoying errors"? what about boring errors? or errors which "pissed me off"? or other types of errors? :) Is it better to avoid of such judgement definitions?

Reply:
1. No. Someone may contribute to this list with other system. 
2. Will do
3. I think current words are fine.